src/main/frontend/components/editor.cljs

```diff
@@ -483,14 +483,17 @@
         ;; FIXME: for translateY layer
         x-overflow-vw? (when (and (seq rect) (> vw-width max-width))
                          (let [delta-width (- vw-width (+ (:left rect) left))]
-                           (< delta-width (* max-width 0.5))))]
+                           (< delta-width (* max-width 0.5))))
+        pos-rect (when (and (seq rect) editing-key)
+                   (:rect (cursor/get-caret-pos (state/get-input))))
+        y-diff (when pos-rect (- (:height pos-rect) (:height rect)))]
```

Comment:
*non-blocking*: for better readability, use `if`

Reply:
For branches with only the truthy condition, it's common clojure practice to use when - https://github.com/bbatsov/clojure-style-guide#when-instead-of-single-branch-if . clj-kondo checks for this by default